Source/Logging.Management/ILogManager.cs

```diff
@@ -12,14 +11,14 @@ namespace Dolittle.Runtime.Logging.Management
     public interface ILogManager
     {
         /// <summary>
-        /// Gets a <see cref="ObservableCollection{T}"/> of <see cref="JsonLogMessage">log messages</see>.
+        /// Gets a <see cref="ObservableCollection{T}"/> of <see cref="string">log messages</see>.
         /// </summary>
-        ObservableCollection<JsonLogMessage> Messages { get; }
+        ObservableCollection<string> Messages { get; }
```

Comment:
We no longer have the JsonLogMessage construct in the fundamentals. This should perhaps be done in a better way, but can we roll with this for now? @einari @jakhog